_events/2017-03-25-testevent.md

```diff
@@ -0,0 +1,7 @@
+---
+date: 25/03/2017
```

Comment:
don’t commit other test events. Only provide one example event where the template is commented. This way I can merge the pull request and not mess up the production site.